src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery should not extend ColorSupplier, instead create an instance of ColorSupplier and use it in the getRandomBall() method. Example: private final ColorSupplier colorSupplier = new ColorSupplier();